openid-4-verifiable-presentations-1_0.md

```diff
@@ -261,6 +268,13 @@ This specification defines the following new parameters:
 
 A public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)). It MAY be passed by the Verifier using the `jwks` or the `jwks_uri` claim within the `client_metadata` or `client_metadata_uri` request parameter.
 
+`request_uri_method`: 
+: OPTIONAL. A string determining the HTTP method to be used with the `request_uri` included in the same request. Two case-insensitive values are defined for `request_uri_method` in this specification: `get` and `post`. If `request_uri_method` is `get`, the Wallet MUST send the request to retrieve the Request Object using the HTTP GET method, i.e., as defined in [@RFC9101]. If `request_uri_method` is `post`, a supporting Wallet MUST send the same request using the HTTP POST method as detailed in (#request_uri_method_post). 
```

Comment:
```suggestion
: OPTIONAL. A string determining the HTTP method to be used with the `request_uri` included in the same request. Two values are defined for `request_uri_method` in this specification: `get` and `post`. If `request_uri_method` is `get`, the Wallet MUST send the request to retrieve the Request Object using the HTTP GET method, i.e., as defined in [@RFC9101]. If `request_uri_method` is `post`, a supporting Wallet MUST send the same request using the HTTP POST method as detailed in (#request_uri_method_post). 
```

Reply:
Nit: The HTTP spec states "The method token is case-sensitive (...). By convention, standardized methods are defined in all-uppercase US-ASCII letters" - https://www.rfc-editor.org/rfc/rfc9110.html#section-9. 
The IANA registry at https://www.iana.org/assignments/http-methods/http-methods.xhtml has `GET` and `POST` so I would suggest that we also use uppercase here.

Reply:
I can accept either upper or lower case, but since parameters in this context are usually lower case, I have a preference for that.